cmake/features.cmake

```diff
@@ -204,4 +206,8 @@ if (ENABLE_SNIPPETS_DEBUG_CAPS)
     add_definitions(-DSNIPPETS_DEBUG_CAPS)
 endif()
 
+if (ENABLE_SNIPPETS_LIBXSMM_TPP)
+    add_definitions(-DSNIPPETS_LIBXSMM_TPP)
```

Comment:
please, don't use global definitions. Move it to plugin

Reply:
Originally, I made it global because it was used both in cpu plugin and in snippets.
But you are right, we should not rely on this flag in the snippets common part.
Moved to the plugin CMake as you suggested.